vocab/security/vocabulary.yml

```diff
@@ -237,7 +242,7 @@ property:
 
   - id: publicKeyJwk
```

Comment:
Expecting to see `secretKeyJwk` defined in this vocabulary as well... otherwise how will people understand how to use it?

Reply:
Yes, secretKeyJwk is added to the vocab in PR https://github.com/w3c/vc-data-integrity/pull/148

Reply:
Definitions added in PR #152